lib/config/config-validator.js

```diff
@@ -97,7 +101,10 @@ function validateRuleOptions(id, options, source) {
 
         if (!validSeverity) {
             message.push(
-                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed \"", severity, "\").\n");
+                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed `",
+                util.inspect(severity).replace(/'/g, "\"").replace(/\n/g, ""),
```

Comment:
another option is just omitting backticks.

```
(you passed "3")
(you passed 3)
(you passed "errrrror")
(you passed [ "errrrror" ])
(you passed [Function: fn])
(you passed true)
```

But the message `you passed true` (without any quotes) seems confusing. It sounds like "you passed correctly".